extensions/wikia/AssetsManager/AssetsManager.class.php

```diff
@@ -46,8 +46,8 @@ private function loadConfig() {
 	 */
 	public static function getInstance() {
 		if( self::$mInstance == false ) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

[project-id]: wikia:app